cmd/outputs.go

```diff
@@ -12,18 +12,21 @@ import (
 // Outputs command retrieves outputs from a workspace (block+env)
 var Outputs = func() *cli.Command {
 	return &cli.Command{
-		Name:      "outputs",
-		Usage:     "Retrieve outputs",
-		UsageText: "nullstone outputs [--stack=<stack-name>] --block=<block-name> --env=<env-name> [options]",
+		Name:        "outputs",
+		Description: "Print all the module outputs for a given block and environment. Provide the `--sensitive` flag to include sensitive outputs in the results. For less information in an easier to read format, use the `--plain` flag.",
```

Comment:
It may helpful to highlight that a user needs access for `--sensitive` to work properly.

Reply:
Added some text to the description for this.